### PR TITLE
Table counts by storage

### PR DESCRIPTION
**Pull request by dipankarkush-db**
### Linked issues

Resolves #..

### Functionality 

- [ ] added relevant user documentation
- [ ] added new CLI command
- [ ] modified existing command: `databricks labs ucx ...`
- [ ] added a new workflow
- [ ] modified existing workflow: `...`
- [ ] added a new table
- [ ] modified existing table: `...`

### Tests

- [ ] manually tested
- [ ] added unit tests
- [ ] added integration tests
- [ ] verified on staging environment (screenshot attached)

### CODE REVIEW

**Comment by dipankarkush-db** [comment]
Screenshot of Assessment (main) dashboard -

image

File: src/databricks/labs/ucx/config.py [hunk 1]
```diff
@@ -94,8 +94,7 @@ class _Config(Generic[T]):
 
     @classmethod
     @abstractmethod
-    def from_dict(cls, raw: dict[str, Any]) -> T:
```
Comment (nfx) [inline src/databricks/labs/ucx/config.py:97]: These changes are irrelevant to this PR, revert them and create a separate PR if you want them in

File: src/databricks/labs/ucx/queries/assessment/main/00_6_count_table_storage_types.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+FROM (
+SELECT
+       CASE
+           WHEN STARTSWITH(location, "dbfs:/mnt") THEN "DBFS MOUNT"
+           WHEN STARTSWITH(location, "/dbfs/mnt") THEN "DBFS MOUNT"
+           WHEN STARTSWITH(location, "dbfs:/databricks-datasets") THEN "Databricks Demo Dataset"
+           WHEN STARTSWITH(location, "/dbfs/databricks-datasets") THEN "Databricks Demo Dataset"
+           WHEN STARTSWITH(location, "dbfs:/") THEN "DBFS ROOT"
+           WHEN STARTSWITH(location, "/dbfs/") THEN "DBFS ROOT"
+           WHEN STARTSWITH(location, "wasb") THEN "UNSUPPORTED"
```
Comment (nfx) [inline src/databricks/labs/ucx/queries/assessment/main/00_6_count_table_storage_types.sql:13]: Why do you skip unsupported? It'll give a false impression to dashboard user

**Review by nfx — CHANGES_REQUESTED** [review]
Remove irrelevant changes from this pr

File: src/databricks/labs/ucx/config.py (hunk 1 above)
Comment (dipankarkush-db) [reply src/databricks/labs/ucx/config.py:97]: I re cloned and only committed changes to the query files relevant for this PR.

File: src/databricks/labs/ucx/queries/assessment/main/00_6_count_table_storage_types.sql (hunk 2 above)
Comment (dipankarkush-db) [reply src/databricks/labs/ucx/queries/assessment/main/00_6_count_table_storage_types.sql:13]: Removed the filter condition.

**Review by dmoore247 — COMMENTED** [review]
Kindly update the assessment documentation describing the widget(s):
https://github.com/databrickslabs/ucx/blob/main/docs/assessment.md

**Comment by dipankarkush-db** [comment]
Moved to a new PR - https://github.com/databrickslabs/ucx/pull/852.

**Pull request closed without merging**